.drone.yml

```diff
@@ -59,7 +57,7 @@ pipeline:
     environment:
       - NIGHTLYBUILD=yes
     commands:
-      - ./project/scripts/sbtPublish ";dotty-bootstrapped/publishSigned ;sonatypeRelease"
+      - ./project/scripts/sbtPublish dotty-bootstrapped/publishSigned sonatypeRelease
```

Comment:
`project/script/sbtPublish` also expects one argument. I am fine with changing it